spec/views/users_index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.feature 'UsersIndices', type: :feature do
+  let!(:user) { User.find(1) }
+
+  def image_exists?(url)
+    response = Net::HTTP.get_response(URI.parse(url))
+    response.code == '200'
+  rescue StandardError
+    false
+  end
+
+  it 'displays the user name and profile image' do
+    visit '/users'
+
+    profile_link = page.find("a[href='#{user_path(user.id)}']", text: 'View_details')
+    profile_link.click
+
+    expect(page).to have_content(user.name)
+    expect(image_exists?(user.photo)).to be true
+    expect(page).to have_content("Number of posts: #{user.posts_counter}")
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/233175092-377a6cbb-3544-4caf-ba90-e7f81b6eb73c.png)

- [ ] It would be nice if you make all your tests pass to verify that your app is working as expected :+1: